recipes/recipes_emscripten/protobuf/recipe.yaml

```diff
@@ -0,0 +1,13 @@
+context:
+  version: 4.22.3
+
+package:
+  name: protobuf
+  version: "{{ version }}"
+
+build:
+  number: 0
+
+source:
+  - url: https://files.pythonhosted.org/packages/e2/86/44a1e4990a81cb4248a2091a182bb76a6417fddcaff560ceb6b44eb05c55/protobuf-{{ version }}.tar.gz
+    sha256: 23452f2fdea754a8251d0fc88c0317735ae47217e0d27bf330a30eec2848811a
```

Comment:
should we add `about` with `license` as a follow up PR? https://github.com/conda-forge/protobuf-feedstock/blob/main/recipe/meta.yaml#L56-L67